BingMaps/spatial-data-services/public-data-sources/poi-entity-types.md

```diff
@@ -11,7 +11,15 @@ ms.service: "bing-maps"
 
 # POI Entity Types
 
-[!INCLUDE [bing-maps-enterprise-service-retirement](../../includes/bing-maps-enterprise-service-retirement.md)]
+> [!NOTE]
+> **Bing Maps Spatial Data Service feature retirement – Points of Interest**
+>
+> The Bing Maps **Spatial Data Service – Points of Interest** is deprecated and will be retired on June 30, 2026. To avoid service disruptions, please update your application by this date. Azure Maps offers this capability with enhanced security, privacy, and compliance features. For Point of Interest data, we recommend using the Azure Maps [Get Search POI](/rest/api/maps/search/get-search-poi) API. Resources to aid in your migration:
```

Comment:
"The Bing Maps Spatial Data Service – Points of Interest data source is deprecated and will be retired on June 30, 2026. Azure Maps offers this capability with enhanced security, privacy, and compliance features. For Point of Interest data, we recommend updating your application to use the Azure Maps [Get Search POI](https://github.com/rest/api/maps/search/get-search-poi) API before that date to avoid service disruptions. We have the following resources available to aid in your migration:"

Reply:
There are multiple sentence fragments and no clear statement to move. I don't think others are chopped up this much.